bench.js

```diff
@@ -138,6 +154,14 @@ suite.add('compile-json-stringify obj', function () {
   CJSStringify(obj)
 })
 
+suite.add('compile-json-stringify date format', function () {
```

Comment:
Please keep consistency in the order of benchmarks and add `JSON.stringify`. Order should be
- JSON.stringify
- FJS
- CJS

Reply:
Added JSON.stringify test and changed order